compendium/LogServiceImpl/src/LogServiceImpl.cpp

```diff
@@ -0,0 +1,100 @@
+#include <sstream>
+
+#include "spdlog/sinks/stdout_color_sinks.h"
+#include "spdlog/spdlog.h"
+
+#include "cppmicroservices/ServiceReference.h"
+
+#include "LogServiceImpl.hpp"
+
+namespace cppmicroservices {
+namespace logservice {
+std::string GetExceptionMessage(const std::exception_ptr& ex)
+{
+  std::string message = "\nException logged: ";
+  if (ex) {
+    std::ostringstream stream;
+    try {
+      std::rethrow_exception(ex);
+    } catch (const std::exception& e) {
```

Comment:
Not sure a stringstream buys us anything here. I think just doing “message += typeid(e).name() + e.what()” would be fine.